cmd/influxd/run.go

```diff
@@ -121,6 +122,26 @@ func (cmd *RunCommand) Run(args ...string) error {
 	return nil
 }
 
+// CheckConfig validates the configuration
+func (cmd *RunCommand) CheckConfing() {
+	if !(cmd.config.Data.Enabled || cmd.config.Broker.Enabled) {
+		cmd.Logger.Fatal("Broker and/or data options must be enabled.  Run `influxd config` for valid config options.")
+	}
+
+	if cmd.config.Broker.Enabled {
+		if cmd.config.Broker.Dir == "" {
+			cmd.Logger.Fatal("Broker.Dir must be specified.  Run `influxd config` for valid config options.")
```

Comment:
Regarding `influxd config` I would write it out fully, as shown above. But this is purely a style question, you may disagree.